docs/chapters/incremental_indexing/index.md

```diff
@@ -0,0 +1,29 @@
+# Incremental Indexing in Jina
+
+## Generating Documents Ids:
+
+In Jina, the document ID is by default generated deterministically by computing a hash the document content. This avoids repeating the doc id while reindexing by using a Random or Incremental counter. The user can `override` document ids with Jina client and add `tags` to map to their unique concepts.
```

Comment:
I would comment on the limitations of custom doc Ids
Reference:
```
       default_logger.critical('Customized ``id`` is only acceptable when: \
            - it only contains the symbols "0"–"9" to represent values 0 to 9, \
            and "A"–"F" (or alternatively "a"–"f"). \
            - it has an even length.')
```

Reply:
done